File: src/main/java/edu/berkeley/bid/bullet/MouseEventsData.java
package edu.berkeley.bid.bullet;

import java.io.Serializable;

public class MouseEventsData implements Serializable {
    private static final long serialVersionUID = 354289467582653L;

    public int m_numMouseEvents;
    public MouseEvent [] m_mouseEvents;

    public MouseEventsData() {
    }

    public MouseEventsData(int n) {
	m_numMouseEvents = n;
	m_mouseEvents = new MouseEvent[n];
    }

    @Override
    public String toString () {
    	return "MouseEventsData[" + m_numMouseEvents + "]";
    }

}

File: src/main/scala/BIDMach/rl/environments/bullet/MouseEventsData.scala
package BIDMach.rl.environments.bullet;

@SerialVersionUID(-24382346314932617L)
class MouseEventsData(n:Int) extends edu.berkeley.bid.bullet.MouseEventsData(n) {
    def this() = this(0);

    def this(k:edu.berkeley.bid.bullet.MouseEventsData) = {
	this(0);
	m_numMouseEvents = k.m_numMouseEvents;
	m_mouseEvents = k.m_mouseEvents;
    };
}
